kernel/comps/virtio/src/device/console/device.rs

```diff
@@ -18,8 +18,8 @@ use crate::{
 pub struct ConsoleDevice {
     config: SafePtr<VirtioConsoleConfig, IoMem>,
     transport: Box<dyn VirtioTransport>,
-    receive_queue: SpinLock<VirtQueue>,
-    transmit_queue: SpinLock<VirtQueue>,
+    receive_queue: SpinLock<VirtQueue<2>>,
+    transmit_queue: SpinLock<VirtQueue<2>>,
```

Comment:
Maybe it's better to use a constant variable instead of a magic number.

Reply:
Yes, this commit just modifies the `VittQueue`, but I can fix it together.